### ISSUE TITLE
Error on windows with python 3.7.3

### ISSUE THREAD

**Issue by Olimelan** [jvillagomez/rssi_module#3]
Hi,

i'm trying to run your module on windows with python 3.7.3 but i always get that error :

File "C:\Users\dk6166\AppData\Local\Programs\Python\Python37-32\lib\subprocess.py", line 1178, in _execute_child startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified

this is my code :

![image](https://user-images.githubusercontent.com/50955088/58256748-8b984500-7d3d-11e9-89c0-537228d44728.png)

Can you help me (i renamed my wifi interface by 'wifi')

thank you

**Comment by TheRizzock** [comment]
I dont believe wifi is the correct interface for windows, but these are [See here](https://superuser.com/questions/251689/is-there-an-iwlist-equivalent-for-windows-xp-or-later). Need to determine how to modify [this method](https://github.com/jvillagomez/rssi_module/blob/8f0a3792aad3334f71ca0f1688f4d8c48bd2ba67/rssi/__init__.py#L54), since it still calls `iwlist ` which is linux specific

**Comment by jvillagomez** [comment]
Hi @Olimelan , as @dkowalsky7 stated, this will not work for Windows. If you found a workaround or modification, please open a PR! I'd love to see this become available for OS outside of Ubuntu.

I have a few new features that I will be implementing over the next few days. Take a look at the README to see a few. :et me know if you'd like to see something else on there.
